stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -569,6 +570,7 @@
                           (pox-addr (tuple (version (buff 1)) (hashbytes (buff 32))))
                           (start-burn-ht uint)
                           (lock-period uint)
+                          (signer-sig (buff 65))
                           (signer-key (buff 33)))
```

Comment:
Thoughts on removing the `signer-key` arg? Seems redundant now that `signer-sig` is passed and the key can be derived from it? I'm guessing the burn-op versions of these functions will do that to save OP_RETURN space.

Reply:
Interesting thought. If it weren't for the burn op issue I'd probably push back, because it's too easy for a mis-created signature to cause an invalid signing key to be used. If the message hash differs, then the signer-key will end up as something else.